requirements/requirements.standalone.txt

```diff
@@ -36,48 +56,97 @@ bracex==2.3.post1
 certifi==2022.5.18.1
     # via requests
 cffi==1.15.0
-    # via cryptography
+    # via
+    #   cryptography
+    #   pycares
 charset-normalizer==2.0.12
-    # via requests
+    # via
+    #   aiohttp
+    #   requests
+click==8.1.3
+    # via pulpcore
 commonmark==0.9.1
     # via rich
 cryptography==37.0.2
     # via
     #   ansible-core
+    #   pulpcore
     #   pyjwt
     #   social-auth-core
 defusedxml==0.7.1
     # via
+    #   odfpy
     #   python3-openid
     #   social-auth-core
+deprecated==1.2.13
+    # via redis
+diff-match-patch==20200713
+    # via django-import-export
 distro==1.7.0
     # via bindep
 django==3.2.13
     # via
     #   django-automated-logging
+    #   django-currentuser
+    #   django-filter
+    #   django-guid
+    #   django-import-export
+    #   django-lifecycle
     #   django-picklefield
     #   djangorestframework
+    #   drf-nested-routers
     #   drf-spectacular
+    #   pulpcore
 django-automated-logging==6.1.3
     # via galaxy-ng (setup.py)
+django-currentuser==0.5.3
+    # via pulpcore
+django-filter==22.1
+    # via pulpcore
+django-guid==3.3.0
+    # via pulpcore
+django-import-export==2.8.0
+    # via pulpcore
```

Comment:
@newswangerd these and other dependencies generate with `pip-compile` when using standard requirements format, but not with `_format_pulp_requirement()`

Reply:
In the `feature/rbac-roles` branch while `requirements.standalone.txt` does not have for example `django-currentuser` and `django-filter` ... after doing a `./compose up` in the developer environment a `pip freeze` shows these in fact are installed, so this is probably fine for development, but we will need to correct this if we want to use `_format_pulp_requirement()` to pin to revisions on `master` for CRC deploys

Reply:
hmmm, that's really disappointing :/

I guess we'll just have to document how to run pulp from a commit and hope people get it right